systers_portal/templates/account/password_change.html

```diff
@@ -0,0 +1,21 @@
+{% extends "account/base.html" %}
+{% load url from future %}
+{% load i18n %}
+{% load crispy_forms_tags %}
+{% block head_title %}{% trans "Change Password" %}{% endblock %}
+{% block content %}
+	<div class="mt40"></div>
+	<div class="container">
+		<form method="POST" id="password-change_form" action="{% url 'account_change_password' %}" class="login form-signup">
```

Comment:
Same problem with CSS class. See your previous PR for details and explanation on what you should do.